rskj-core/src/main/java/co/rsk/peg/bitcoin/FlyoverRedeemScriptCreationException.java

```diff
@@ -0,0 +1,17 @@
+package co.rsk.peg.bitcoin;
+
+public class FlyoverRedeemScriptCreationException extends RuntimeException {
```

Comment:
Maybe simply reuse RedeemScriptCreationException? I think it covers what we need

Reply:
done